src/app/settings/settings.component.ts

```diff
@@ -1,10 +1,13 @@
 import { Component, NgZone, OnDestroy, OnInit } from "@angular/core";
 
+import { ApiService } from "jslib-common/abstractions/api.service";
 import { BroadcasterService } from "jslib-common/abstractions/broadcaster.service";
 import { OrganizationService } from "jslib-common/abstractions/organization.service";
 import { PlatformUtilsService } from "jslib-common/abstractions/platformUtils.service";
 import { TokenService } from "jslib-common/abstractions/token.service";
 
+import { StateService } from "src/abstractions/state.service";
```

Comment:
It is! Just a bit confusing because this is actually a web specific `StateService` that allows for use of additional abstracted methods. It extends our `jslib` `StateService` abstraction within.